.env.example

```
@@ -31,48 +31,14 @@ LOGTO_MANAGEMENT_APP_SECRET=your_logto_management_app_secret_here
-
-QUESTDB_HOST=questdb
-QUESTDB_PORT=8812
-QUESTDB_USER=admin
-QUESTDB_PASSWORD=change_me_for_local_pgwire
-QUESTDB_DB=qdb
-QUESTDB_HTTP_USER=admin
-QUESTDB_HTTP_PASSWORD=change_me_for_local_http
+PG_HOST=
+PG_PORT=5432
+PG_USER=
+PG_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PG_PASSWORD key should go before the PG_PORT key

(UnorderedKey)